les/protocol.go

```diff
@@ -34,17 +34,18 @@ import (
 const (
 	lpv2 = 2
 	lpv3 = 3
+	lpv4 = 4
 )
 
 // Supported versions of the les protocol (first is primary)
 var (
-	ClientProtocolVersions    = []uint{lpv2, lpv3}
-	ServerProtocolVersions    = []uint{lpv2, lpv3}
+	ClientProtocolVersions    = []uint{lpv2, lpv3, lpv4}
+	ServerProtocolVersions    = []uint{lpv2, lpv3, lpv4}
```

Comment:
Are you sure we should bump the protocol version in this PR? I want to add at least the tx unindexing fix to les/4 and we should not have a half implemented and enabled protocol version on master. If you want to merge it after my PR then it's fine but this PR is otherwise ready to merge. If you want to merge it now then lpv4 should not be added to the supported list yet.